tasks/verify-enterprise-contract.yaml

```diff
@@ -26,7 +26,7 @@ spec:
 
     - name: COSIGN_EXPERIMENTAL
       type: string
-      description: Control transparency log lookups. Set to "1" to enabled it.
+      description: Control transparency log lookups. Set to "0" to enabled it.
```

Comment:
Changing the description to mention the opposite value than what is currently in default.

Reply:
Should be `to disable it` iiuc.

Reply:
This one should be a blocker, otherwise it's some half-baked idea sharing...